### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by machadovilaca**
This PR updates the main README file with instructions on how to run the service with the `lvctl` tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,41 +14,51 @@ The query will be bound by the Virtual Machine lifecycle timeline.
...
+[lvctl documentation](./tools/lvctl/README.md).
+
+### Get a lvctl binary
 
-- creating an instance
 ```bash
+  # To get the latest released lvctl binary
+  LATEST_TAG=$(curl https://api.github.com/repos/kubevirt/kubevirt/releases/latest | grep -i "tag_name" | awk -F '"' '{print $4}')
+  wget https://github.com/vladikr/logsviewer/releases/download/${LATEST_TAG}/lvctl
+  chmod +x lvctl
```
[the remark this answers is not in the document]
Comment (machadovilaca) [reply README.md:27]: wdyt?

File: README.md [hunk 2]
```diff
@@ -14,41 +14,51 @@ The query will be bound by the Virtual Machine lifecycle timeline.
...
 ```
 
-- delete the instance
+This commands will log the instance id and the route created to access the
+LogsViewer UI.
+
+> **Warning**
+> By default, the instance will be deleted 48 hours after the last must-gather
+> file was imported. Or after creation time if no must-gather file was imported.
+> Check the '-deletion-condition' and '-deletion-delay' flags for more details.
```
Comment (vladikr) [inline README.md:50]: Maybe we should also mention how can one opt out of this.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vladikr) [reply README.md:27]: Yeah. Let's see how do we do this.

File: README.md (hunk 2 above)
Comment (vladikr) [reply README.md:50]: @machadovilaca we should specify how to check that.
Also, we probably need a --help flag for the lvctl tool

**Pull request merged**